OpenRA.Mods.Common/UtilityCommands/DumpSequenceSheetsCommand.cs

```diff
@@ -50,7 +55,9 @@ void IUtilityCommand.Run(Utility utility, string[] args)
 			{
 				var max = s == sb.Current ? (int)sb.CurrentChannel + 1 : 4;
 				for (var i = 0; i < max; i++)
-					s.AsPng((TextureChannel)ChannelMasks[i], palette).Save($"{count++}.png");
+					s.AsPng((TextureChannel)ChannelMasks[i], palette).Save($"{count}.{i}.png");
```

Comment:
```suggestion
					s.AsPng((TextureChannel)ChannelMasks[i], palette).Save($"{count++}.{i}.png");
```

Without this it feels weird. It logs `Saved [0..2].png` but it's actually many more images

<img width="242" alt="Screenshot 2023-03-08 at 12 14 50" src="https://user-images.githubusercontent.com/37534529/223685849-ed33d16d-112f-410b-8a1c-adba41827dba.png">

What exactly is the idea behind this change?

Reply:
If this is intended. Then perhaps another entry mentioning how many images?